app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageExpiration.kt

```diff
@@ -202,6 +203,7 @@ class SelfDeletionTimer(private val context: Context) {
 
             fun decreaseTimeLeft(interval: Duration) {
                 if (timeLeft.inWholeSeconds != 0L) timeLeft -= interval
+                Log.d("TEST", "time left ${timeLeft.inWholeSeconds}")
```

Comment:
Should this log be here? If so could we change the `Log` to `appLogger` and add the class tag?